Setup/Assets/Languages/Arabic, Egypt.iglang

```diff
@@ -240,6 +240,7 @@
       <Item key="frmSetting.lblToolbarPosition" value="موضع شريط الأدوات:"/>
       <Item key="frmSetting.cmbToolbarPosition._Top" value="أعلى"/>
       <Item key="frmSetting.cmbToolbarPosition._Bottom" value="أسفل"/>
+      <Item key="frmSetting.lblToolbarIconHeight" value="Toolbar icon size:"/>
```

Comment:
Don't need to update the language pack. When I build for the final release, I will update from Crowndin.